checkout/application/orderService.go

```diff
@@ -295,16 +349,6 @@ func (os *OrderService) placeOrderWithPaymentProcessing(ctx context.Context, dec
 			CreditCardInfo:  transaction.CreditCardInfo,
 		})
 	}
```

Comment:
If i see correctly the cart is only used to fetch the e-mail? Should we not add the cart to the PlacedOrderInfo?